src/algebra/order/monoid_lemmas_zero_lt.lean

```diff
@@ -126,18 +126,19 @@ end abbreviations_mono
 
 variables {a b c d : α}
 
+
```

Comment:
Why are you adding these spaces? They're not in the style conventions, right?

Reply:
Oh, I will remove it.